pkg/lca/imagebasedupgrade.go

```diff
@@ -223,6 +223,22 @@ func (builder *ImageBasedUpgradeBuilder) WithSeedImage(
 	return builder
 }
 
+// WithOadpContent adds oadpContent to be used by the imagebasedupgrade.
+func (builder *ImageBasedUpgradeBuilder) WithOadpContent(
+	oadpConentName string, oadpConentNamespace string) *ImageBasedUpgradeBuilder {
```

Comment:
WDYT about using parameter names like configMapName and configMapNamespace or some variation instead? It isn't clear that what you are passing in is a configmap reference based on the current argument names.

Reply:
Updated.